SparkAppItem.h

```diff
@@ -1,6 +1,6 @@
 @interface SparkAppItem : NSObject
-    @property (nonatomic, retain) NSString* bundleIdentifier;
-    @property (nonatomic, retain) NSString* displayName;
+@property (nonatomic, retain) NSString* bundleIdentifier;
+@property (nonatomic, retain) NSString* displayName;
 
-    -(UIImage*)icon;
-@end
\ No newline at end of file
+-(UIImage*)icon;
```

Comment:
```suggestion
- (UIImage*)icon;
```

Reply:
Done and also edited the others to match.